KMB/requirements.txt

```diff
@@ -1 +1 @@
-git+https://github.com/lokal-profil/BatchUploadTools.git@0.1.2
+git+https://github.com/lokal-profil/BatchUploadTools.git@0.1.5
```

Comment:
That is 0.1.5 after https://github.com/lokal-profil/BatchUploadTools/pull/28 is merged.